drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/client/ThriftClientHandler.java

```diff
@@ -77,14 +81,15 @@
         extends ChannelDuplexHandler
 {
     private static final int ONEWAY_SEQUENCE_ID = 0xFFFF_FFFF;
+    private static final Logger log = Logger.get(ThriftClientHandler.class);
 
     private final Duration requestTimeout;
     private final Transport transport;
     private final Protocol protocol;
 
     private final ConcurrentHashMap<Integer, RequestHandler> pendingRequests = new ConcurrentHashMap<>();
     private final AtomicReference<TException> channelError = new AtomicReference<>();
-    private final AtomicInteger sequenceId = new AtomicInteger(42);
+    private static final AtomicInteger sequenceId = new AtomicInteger(42);
```

Comment:
Would it cause problem when multiple instances are accessing and changing this value?